com.rtosta.zapzap.yaml

```diff
@@ -0,0 +1,37 @@
+app-id: com.rtosta.zapzap
+runtime: org.kde.Platform
+runtime-version: "6.2"
+sdk: org.kde.Sdk
+base: io.qt.qtwebengine.BaseApp
+base-version: "6.2"
+comand: "zapzap"
+finish-args:
+  - "--share=ipc"
+  - "--socket=fallback-x11"
```

Comment:
you have no wayland socket. so this won't work

Reply:
With Wayland it is failing. I believe I should have just put x11. Correct?